docs/products/postgresql/howto/check-iop.rst

```diff
@@ -0,0 +1,57 @@
+IOP capability of plans
```

Comment:
I am not sure what IOP means here, do you think it makes sense to add the full name?